mlflow/server/js/src/components/ExperimentRunsTableCompactView.js

```diff
@@ -100,7 +100,7 @@ class ExperimentRunsTableCompactView extends PureComponent {
 
 
   /** Returns a row of table content (i.e. a non-header row) corresponding to a single run. */
-  getRow({ idx, isParent, hasExpander, expanderOpen, childrenIds }) {
+  getRow({ idx, isParent, hasExpander, expanderOpen, childrenIds, sortedRunIds, displayIndex }) {
```

Comment:
What does `displayIndex` mean here? Why is it an argument to `getRow`? It seems to ultimately be relevant for `onCheckbox`? I guess it's just the index of this particular row within `sortedRunIds` (is that different from `idx`)? It may be useful to define these parameters explicitly in the JS doc.

Reply:
Yep, it's just the index of the current row within `sortedRunIds` - will clarify this in the JS doc. `idx` is different - it's the index of the current row within the unsorted list of runs returned by the REST API, and is used to get metric/param/tag values associated with the current run. It'd be nice to clean this up & just use the run ID to look up said metric/param/tag values, but will defer it for a follow-up